specs/global/responses.json

```
@@ -322,7 +322,7 @@
         }
     },
     "Statement": {
-        "description": "A single wikibase statement",
+        "description": "A single Wikibase statement",
```

Shouldn't this singular be capitalized as well then?

```suggestion
        "description": "A single Wikibase Statement",
```